zrml/global-disputes/README.md

```diff
@@ -32,11 +32,19 @@ on which the market finally resolves.
   get their reward. Fails if the global dispute is not concluded yet.
 - `reward_outcome_owner` - Reward the collected fees to the owner(s) of a voting
   outcome. Fails if not all outcomes are already purged.
+- `refund_vote_fees` - Return all vote funds and fees, when a global dispute was
+  destroyed.
 
```

Comment:
In the _Terminology_ section, there seems to be a lot of stuff missing. At least I don't think _outcome sum_ is the only thing that needs to be explained.